### ISSUE TITLE
nie działa dodawanie członków do rejonu

### ISSUE THREAD

**Issue by quchasz** [edk-software/rejony-app#157]
próba dodania członka do rejonu:
- kończy się komunikatem że zaproszenie zostało wysłane,
- dochodzi mail z informacją że jest zaproszenie do współpracy,
- po zalogowaniu do panelu zaproszenia nie ma -> nie da się więc dołączyć do rejonu

**Comment by zaquncio** [comment]
@quchasz czy na danego maila konto jest zarejestrowane kilka razy? sprawdzisz to tutaj: https://my.edk.org.pl/admin/user/index

**Comment by quchasz** [comment]
@zaquncio tak, tu jest źródło problemu - zaproszenie wysłane na konto/maila który juz jest w systemie nie generuje klucza:
![Screen Shot 2020-02-21 at 10 33 54](https://user-images.githubusercontent.com/47346557/75022771-f7f3d080-5496-11ea-9129-8bb2ca026c5f.png)
podczas gdy zaproszenie wysłane na maila którego nie ma w systemie taki klucz generuje:
![Screen Shot 2020-02-21 at 10 33 39](https://user-images.githubusercontent.com/47346557/75022829-0fcb5480-5497-11ea-8f79-31b34cb5b377.png)

**Comment by quchasz** [comment]
usunięcie użytkownika (a więc adresu email) i wysłanie ponownego zaproszenia nie rozwiązuje problemu - pojawia się taki błąd:
<img width="239" alt="Screen Shot 2020-02-22 at 18 40 43" src="https://user-images.githubusercontent.com/47346557/75096762-ffa09b80-55a2-11ea-9885-a6b4926f32af.png">

**Comment by quchasz** [comment]
zaproszenia jednak działają prawidłowo, jedyny problem jaki udało się zreprodukować to przypadek kiedy w systemie jest 2 lub więcej użytkowników o tym samym mailu - wówczas zaproszenie idzie na pierwszego ze znalezionych userów
